cli/src/commands/bootstrap_config.py

```diff
@@ -96,7 +100,7 @@ def key_sort(attr):
         "comments": 29,
         "nb_hits": 30
     }
-    if attr[0] in ref.keys():
+    if attr[0] in list(ref.keys()):
```

Comment:
```suggestion
    if attr[0] in ref:
```

Reply:
Not compliant with python 2.7, Will keep it in our backlog when we will stop supporting 2.7. Thanks for sharing.